linking/static/linked-against-cpp-standard-library.yml

```diff
@@ -0,0 +1,16 @@
+rule:
+  meta:
+    name: linked against CPP standard library
```

Comment:
should we use "C++" here?

Reply:
All existing rules use CPP and I think we'd have to update the linter.